cotk/dataloader/dataloader.py

```diff
@@ -215,6 +216,26 @@ def get_batches(self, key, batch_size=None, shuffle=True, ignore_left_samples=Fa
 				break
 			yield res
 
+	def get_all_batch(self, key):
+		r'''Concatenate all batches
+
+        Arguments:
+            key (str): key name of dataset, must be contained in ``self.key_name``.
+
+        Returns:
+            A dict like :func:`get_batch`
+        '''
+		res = {}
+		for batch in self.get_batches(key, batch_size=32):
```

Comment:
Use ``get_batch`` instead of  get_batches, because it cause an side-effect when calling restart